internal/services/kusto/migration/kusto_cluster_migration_v0_to_v1.go

```diff
@@ -148,7 +148,7 @@ func (s KustoAttachedClusterV0ToV1) Schema() map[string]*pluginsdk.Schema {
 		},
 
 		"language_extensions": {
-			Type:     pluginsdk.TypeList,
+			Type:     pluginsdk.TypeSet,
```

Comment:
this is a point-in-time reference which shouldn't change - so this would still be a List:

```suggestion
			Type:     pluginsdk.TypeList,
```